example/index2d.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
```

Comment:
Think it makes more sense to keep different examples in different directories, e.g. create example_2d/ and example_3d/ directories.

This keeps each example more simple, and will scale better if we want to add more examples.